closed/make/CreateJars.gmk

```diff
@@ -86,22 +86,99 @@ RT_JAR_EXCLUDES += \
 	com/ibm/dtfj \
 	com/ibm/java/diagnostics \
 	com/ibm/jvm/Debug.class \
-	com/ibm/jvm/dtfjview \
-	com/ibm/jvm/format \
 	com/ibm/jvm/FormatTimestamp.class \
 	com/ibm/jvm/Indent.class \
 	com/ibm/jvm/InputFile.class \
-	com/ibm/jvm/j9 \
 	com/ibm/jvm/MessageFile.class \
 	com/ibm/jvm/OutputFile.class \
 	com/ibm/jvm/ProgramOption.class \
 	com/ibm/jvm/Statistics.class \
 	com/ibm/jvm/Summary.class \
 	com/ibm/jvm/Threads.class \
 	com/ibm/jvm/Timezone.class \
-	com/ibm/jvm/trace \
-	com/ibm/jvm/traceformat \
 	com/ibm/jvm/TraceFormat.class \
 	com/ibm/jvm/Verbose.class \
+	com/ibm/jvm/dtfjview \
+	com/ibm/jvm/format \
+	com/ibm/jvm/j9 \
+	com/ibm/jvm/trace \
+	com/ibm/jvm/traceformat \
 	com/ibm/tools/attach/attacher \
 	#
+
+OPENJ9_MANAGEMENT_TYPES := \
+	com/ibm/java/lang/management/internal/BufferPoolMXBeanImpl.class \
```

Comment:
Can we use wildcards to include entire packages? A hard code list of classes will likely result in problems when new classes are added.

Reply:
Yes, we ought to be do that.